db/schema.rb

```diff
@@ -317,7 +317,7 @@
     t.string "title", limit: 150, default: "", null: false
     t.text "description", size: :medium
     t.text "body", size: :medium
-    t.timestamp "created_at", default: -> { "current_timestamp() ON UPDATE current_timestamp()" }, null: false
+    t.timestamp "created_at", default: -> { "CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP" }, null: false
```

Comment:
Please revert this unrelated change. When you are preparing your commits, you can run `git add -p` to add specific changed chunks and not these unrelated ones. It looks like you had some hassles getting mariadb running via the trilogy gem?